pages_bots/page.py

```
@@ -50,28 +50,46 @@
-super_page.add_Usertables(User_tables, "wikidata")
-bot_api.add_Usertables(User_tables, "wikidata")
-catdepth_new.add_Usertables(User_tables, "wikidata")
-# ---
-NEW_API = bot_api.NEW_API
-MainPage = super_page.MainPage
 change_codes = lang_codes.change_codes
-CatDepth = catdepth_new.subcatquery
+
+logins_cache = {}
+
+def log_it(lang, family):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)